build/package_release.php

```diff
@@ -156,7 +156,7 @@
 system('zip -r ../packages/'.$appVersion.'.zip . -x@../exclude_files.txt -x@../exclude_files_full.txt > /dev/null');
```

Comment:
While we're at it, let's also optimize this one by doing this instead (see my comment below for more context):

```suggestion
system('zip -qr ../packages/'.$appVersion.'.zip . -x@../exclude_files.txt -x@../exclude_files_full.txt');
```